core/src/main/java/de/danoeh/antennapod/core/syndication/namespace/NSITunes.java

```diff
@@ -67,13 +69,28 @@ public void handleElementEnd(String localName, HandlerState state) {
                 } else {
                     return;
                 }
-
                 state.getTempObjects().put(DURATION, duration);
             } catch (NumberFormatException e) {
                 e.printStackTrace();
             }
+        } else if (localName.equals(SUBTITLE)) {
+            String subtitle = state.getContentBuf().toString();
+            if (state.getCurrentItem() != null) {
+                if (TextUtils.isEmpty(state.getCurrentItem().getDescription())) {
+                    state.getCurrentItem().setDescription(subtitle);
+                }
+            } else {
+                if (TextUtils.isEmpty(state.getFeed().getDescription())) {
+                    state.getFeed().setDescription(subtitle);
+                }
+            }
+        } else if (localName.equals(SUMMARY)) {
```

Comment:
Not really. itunes summary will overwrite the description if we parse the tag after it, but subtitle is only used if the description is currently empty (and will of course be overwritten by summary or description is there is one)